### ISSUE TITLE
Better env diff

### ISSUE THREAD

**Issue by amenzhinsky** [containers/ansible-podman-collections#163]
Any variable change in `env` reports whole vars dictionary including image variables `PATH`, `TERM`, etc., which is not very handy especially where there are many of them.

btw docker_container handles it well.

**Comment by sshnaidm** [comment]
@amenzhinsky a use case or an example would be useful, thanks

**Comment by amenzhinsky** [comment]
```
TASK [Create blah container] ****************************************************
--- before
+++ after
@@ -15,7 +15,7 @@
         "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-        "RELEASE_VERSION=v0.19-117-ge5a6f67",
+        "RELEASE_VERSION=v0.19-118-gb92ece2",
         "ADDR=:6065",
```

**Comment by sshnaidm** [comment]
I think it should be fixed with #745 
Please reopen if it's not the case
